cmd/integration/harness.go

```diff
@@ -85,7 +86,8 @@ func main() {
 	log.With().Info("integration: harness is listening on a blocking dummy channel")
 
 	interruptChannel := make(chan os.Signal, 1)
-	signal.Notify(interruptChannel, os.Interrupt)
+	// os.Interrupt for all systems, especially windows, syscall.SIGTERM is mainly for docker.
+	signal.Notify(interruptChannel, os.Interrupt, syscall.SIGTERM)
```

Comment:
This could also be done with a `context` 🙂 

```golang
	// os.Interrupt for all systems, especially windows, syscall.SIGTERM is mainly for docker.
	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
	defer cancel()
```

Reply:
done